### PR TITLE
[css-in-js] EmptyState widget

### PR DESCRIPTION
**Pull request by grosbouddha**

### CODE REVIEW

**Comment by pith** [comment]
nit: can you add a screenshot of the migrated component ?

File: src/emptyState/EmptyState.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+interface EmptyState {
+  readonly className?: Classcat.Class
+  readonly extraClassName?: Classcat.Class
+  readonly image: string
+  readonly text: string
+  readonly button?: JSX.Element
+}
+
+const EmptyState = ({ className, image, text, button, extraClassName}: EmptyState) => (
+  <div className={cc(['kirk-empty-state', extraClassName])}>
```
Comment (pith) [inline src/emptyState/EmptyState.tsx]: You missed the className here.

File: src/emptyState/story.tsx [hunk 2]
```diff
@@ -4,7 +4,7 @@ import { storiesOf } from '@storybook/react'
 import { withKnobs, text, boolean } from '@storybook/addon-knobs'
 import spec from 'emptyState/specifications/emptystate.md'
 
-import EmptyState from 'emptyState'
+import EmptyState from './index'
```
Comment (pith) [inline src/emptyState/story.tsx]: nit: I think that you don't need to change it.

Comment (grosbouddha) [reply src/emptyState/story.tsx]: I need to get the new styles component in index.jsx

**Pull request merged**